string data
---
int64 count